src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const valuesArray = Object.values(robot);
+  const isDuplicate = new Set(valuesArray).size !== valuesArray.length;
+
+  const robotReverse = {};
+
+  for (const key in robot) {
+    robotReverse[robot[key]] = key;
+  }
+
+  return isDuplicate ? null : robotReverse;
```

Comment:
```suggestion
  const valuesArray = Object.values(robot);
  const isDuplicate = new Set(valuesArray).size !== valuesArray.length;
  
  if (isDuplicate) {
  	return null;
  }

  const robotReverse = {};

  for (const key in robot) {
    robotReverse[robot[key]] = key;
  }

  return robotReverse;
```

don't do unnecessary loop if you gonna return `null` anyway